### ISSUE TITLE
Running on Rasberry Pi 4B 

### ISSUE THREAD

**Issue by patwolfe** [jsandler18/raspi-kernel#26]
I'm trying to get this code running on my RPi4B and I can't seem to get it working. I've changed PERIPHERAL_BASE to 0xFE000000 as documented [here](https://www.raspberrypi.org/forums/viewtopic.php?t=243846#p1498033), but my Raspberry Pi gets stuck on the rainbow screen when booting. Does anyone have advice for further debugging this issue? 

I've installed Raspbian on my SD card, and have deleted the other kernel images and replaced kernel7.img with the image built from the Makefile. On QEMU I'm able to run it fine (as long as I switch the PERIPHERAL_BASE back to the 2/3 address)

**Comment by StrSebastian** [comment]
yeah i have the same issue here. pls write me if u fixed it. To get sure, i installed it on Raspi 1 and 2 but nothing showed on screen:(
@wolfep15
